SAMtags.tex

```diff
@@ -317,11 +320,29 @@ \subsection{Barcodes}
 \subsection{Original data}
 
 \begin{description}
+\item[OA:Z:\tagregex{{\tt (}\sf{RNAME}{\tt,\,}\sf{POS}{\tt,\,}\emph{strand}{\tt,\,}\sf{CIGAR}{\tt,\,}\sf{MAPQ}{\tt,\,}\sf{NM}{\tt; )}+}]
+The tag encodes the original alignment information of the segment prior to realignment or unalignment by subsequent tool. 
```

Comment:
Either change `segment` to `record` if we only support a GATK indel realignment style single record events, or add something along the lines of:
`Realignment operations that realign an entire segment should report the full previous chimeric alignment. Realignment operations that realign a single record should report the previous alignment of only that record.`

Reply:
I think I'll leave it on single records for now...so I'll change segment to record as you suggested.